understanding/22/redundant-entry.html

```diff
@@ -12,7 +12,7 @@ <h1>Understanding Redundant Entry</h1>
         <h2>In brief</h2>
         <dl>
             <dt>Goal</dt><dd>Make it easier for users to complete multi-step processes.</dd>
-            <dt>What to do</dt><dd>Don't ask for the same information twice in the same session.</dd>
+            <dt>What to do</dt><dd>Don't ask for the same information twice in the same process.</dd>
```

Comment:
```suggestion
            <dt>What to do</dt><dd>Don't ask for the same information twice in the same procedure.</dd>
```

Reply:
personally, i find "process" more immediately understandable than "procedure".

also, process is a defined term https://www.w3.org/TR/WCAG22/#dfn-processes whereas "procedure" doesn't seem to feature in WCAG at this point

Reply:
"procedure" just makes my mind go straight to "medical procedure" for some reason...

Reply:
if we do want to avoid the word "process", alternatives perhaps: "activity" or "user flow"? still not sure it would help, or whether that would just throw up more confusion for readers when it's not defined and unambiguous terms